docs/crio.8.md

```diff
@@ -210,6 +211,10 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
        mounted into the container for the volumes.
 	3. ignore: All volumes are just ignored and no action is taken. (default: mkdir)
 
+**--infra-containers-cpus**="": CPU set to run infra containers, if not specified the CRI-O will use all online CPUs to run infra containers (default: "").
+    You can specify CPUs in the Linux CPU list format.
```

Comment:
suggestion: I think `--infra-cpus` would work too
nit: we should describe that these should be the same as the `guaranteed-cpus` passed to the kubelet, if the user wants their workloads to truly have a unique cpuset

Reply:
actually, above we use `infra-ctr` so we should use that here `--infra-ctrs-cpus`

Reply:
done

Reply:
How about `infra-cpuset`?

Reply:
`cpus` is too broad and could also refer to amount of cpu shares reserved or cpu quota if we want to add those in the future. So, I think it may be better to be explicit.

Reply:
makes sense, done